ner_v2/detectors/numeral/number/standard_number_detector.py

```diff
@@ -21,18 +24,21 @@ def __init__(self, entity_name, data_directory_path, min_digit, max_digit):
         self.original_date_text = []
         self.entity_name = entity_name
         self.tag = '__' + entity_name + '__'
-        self.min_digit = min_digit
-        self.max_digit = max_digit
 
         self.numbers_word = {}
-        self.language_number_map = {}
+        self.language_scale_map = {}
+        self.units_map = {}
 
         # Method to initialise value in regex
         self.init_regex_and_parser(data_directory_path)
 
+        self.regex_numeric_patterns = re.compile(r'(([\d,]+[.]?[\d]*)\s?(' + "|".join(self.language_scale_map.keys())
```

Comment:
Can be, then it will be compiled each time called for word

Reply:
members of `self.language_scale_map.keys()` should probably be pass through `re.escape` before putting them into a regex pattern

Reply:
This pattern would not capture `.5`, `.25`, etc decimals with missing 0

Reply:
Given the space between digits and scale is optional. This would detect something like `twothousand` with a missing space. I don't know if that is good or bad

Reply:
I am not sure, if we should capture these patterns. Also `.5`  is this be considered as `0.5` or `5`. Might be it comes after a full stop. What your views here?